package.json

```diff
@@ -32,20 +32,20 @@
     "@types/sinon": "^4.0.0",
     "chai": "^4.1.2",
     "del": "^3.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
     "gulp-mocha": "^3.0.1",
     "gulp-replace": "^0.6.1",
     "gulp-shell": "^0.6.3",
     "gulp-sourcemaps": "^2.6.1",
     "gulp-tslint": "^8.1.2",
     "gulp-typescript": "^3.2.3",
-    "gulpclass": "^0.1.2",
+    "gulpclass": "^0.2.0",
     "mocha": "^3.2.0",
     "reflect-metadata": "^0.1.10",
     "sinon": "^4.1.2",
     "sinon-chai": "^2.14.0",
     "ts-node": "^3.3.0",
-    "tslint": "^5.8.0",
+    "tslint": "5.8.0",
```

Comment:
A patch fix version of tslint is causing a newer version of `@babel/code-frame` to get loaded which is incompatible with node v4. Locking down to the earlier version so it will successfully run in v4.

Reply:
PR #76 is failing for this same reason.